### PR TITLE
EE-21594, EE-21768 - Added readme and feedback CRUD for Java SDK

### PR DESCRIPTION
**Pull request by gmarginean-sdl**

### CODE REVIEW

File: java/src/main/java/com/languageweaver/sdk/samples/feedback/EdgeFeedbackService.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                    .setSourceLanguageId("eng")
+                    .setTargetLanguageId("fra")
+                    .setModel("generic")
+                    .setSourceText("sourceText")
+                    .setTargetText("targetText")
+                    .setSuggestedTranslation("suggestedTranslation")
+                    .setComment(FeedbackComment.SPELLING);
+
+            feedback = lwClient.createFeedback(createFeedbackRequest);
+            System.out.println(feedback.getFeedbackId());
```
Comment (LauraScurtu) [inline java/src/main/java/com/languageweaver/sdk/samples/feedback/EdgeFeedbackService.java:35]: Same here

File: java/src/main/java/com/languageweaver/sdk/samples/feedback/CloudFeedbackService.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                    .setSuggestedTranslation("suggestedTranslation")
+                    .setRating(3)
+                    .addComment(FeedbackComment.SPELLING)
+                    .addComment("custom comment")
+                    .setTranslationMetadata("metadata")
+                    .setNumberOfSegments(1)
+                    .setQualityEstimationMT(QualityEstimation.ADEQUATE);
+
+            feedback = lwClient.createFeedback(createFeedbackRequest);
+            System.out.println(feedback.getFeedbackId());
```
Comment (LauraScurtu) [inline java/src/main/java/com/languageweaver/sdk/samples/feedback/CloudFeedbackService.java:37]: I would create separate methods for each operation.

File: java/src/main/java/com/languageweaver/sdk/samples/feedback/FeedbackService.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            CreateFeedbackRequest createFeedbackRequest = new CreateFeedbackRequest()
+                    .setSourceLanguageId("eng")
+                    .setTargetLanguageId("fra")
+                    .setModel("generic")
+                    .setSourceText("sourceText")
+                    .setTargetText("targetText")
+                    .setSuggestedTranslation("suggestedTranslation")
+                    .addComment(FeedbackComment.SPELLING);
+            feedback = lwClient.createFeedback(createFeedbackRequest);
+            System.out.println(feedback.getFeedbackId());
```
Comment (LauraScurtu) [inline java/src/main/java/com/languageweaver/sdk/samples/feedback/FeedbackService.java:29]: same

File: dotnet/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+## .Net SDK samples
```
Comment (LauraScurtu) [inline dotnet/README.md:1]: Add this information to the general readme.
We want to be displayed on this page: https://github.com/RWS/languageweaver-sdk-samples/.

File: java/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+## Java SDK samples
```
Comment (LauraScurtu) [inline java/README.md:1]: Add this information to the general readme.
We want to be displayed on this page: https://github.com/RWS/languageweaver-sdk-samples/.

**Review by LauraScurtu — CHANGES_REQUESTED** [review]

**Review by alungu99 — CHANGES_REQUESTED** [review]

File: java/src/main/java/com/languageweaver/sdk/samples/feedback/CloudFeedbackService.java [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+            } while (filteredFeedback.getFeedbackList().size() > 0);
+        }
+    }
+
+    public static void main(String[] args) throws IOException, ParseException {
+        try (CloudLanguageWeaverClient lwClient = new CloudLanguageWeaverClient().build()) {
+            CloudFeedbackResult feedback;
+
+//          CREATE FEEDBACK
+            feedback = createFeedback();
```
[the remark this answers is not in the document]
Comment (gmarginean-sdl) [reply java/src/main/java/com/languageweaver/sdk/samples/feedback/CloudFeedbackService.java:120]: Normally yes but for the purpose of this sample I think it is ok to have the full code in the methods.

File: java/src/main/java/com/languageweaver/sdk/samples/dictionaries/CloudDictionaryService.java [hunk 7]
```diff
@@ -1,7 +1,7 @@
 package com.languageweaver.sdk.samples.dictionaries;
 
-import com.languageweaver.sdk.CloudLanguageWeaverClient;
-import com.languageweaver.sdk.translate.result.cloud.CloudDictionariesResult;
+import com.languageweaver.sdk.common.cloud.CloudLanguageWeaverClient;
+import com.languageweaver.sdk.translate.cloud.result.CloudDictionariesResult;
```
Comment (alungu99) [inline java/src/main/java/com/languageweaver/sdk/samples/dictionaries/CloudDictionaryService.java:5]: please resolve conflicts

**Review by alungu99 — CHANGES_REQUESTED** [review]

**Comment by gmarginean-sdl** [comment]
Created another [pull request](https://github.com/RWS/languageweaver-sdk-samples/pull/35) for these changes.

**Pull request closed without merging**